source/whats-new.txt

```diff
@@ -45,6 +43,82 @@ What's New in 1.12
      deprecated. You can create a client and connect in one call
      by using the ``mongo.Connect()`` method.
 
+New features of the 1.12 Go driver release include: 
+
+Queryable Encryption
+~~~~~~~~~~~~~~~~~~~~
+
+The release adds production-ready support for
+:ref:`qe-manual-feature-qe` (QE). To use
+the QE feature, you must use MongoDB Server version 7.0 or later and
+``libmongocrypt`` version 1.8.0 or later.
```

Comment:
@ccho-mongodb I agree it could be confusing since MongoDB 7.0 isn't GA yet. Should I also make a similar change to the Github and community forum release notes?

Reply:
@matthewdale If the same text is used in the release notes, I'd probably omit "production-ready support" and let the users discover that the 7.0 RC would be the risk factor in putting a QE implementation to production rather than explicitly mentioning it since eventually 7.0 will be released.